lamindb/core/_sync_git.py

```diff
@@ -68,9 +65,8 @@ def get_git_commit_hash(blob_hash: str, repo_dir: Path | None = None) -> str | N
     if commit_hash == "" or result.returncode == 1:
         return None
     else:
-        assert (
-            len(commit_hash) == 40
-        ), f"commit hash |{commit_hash}| is not 40 characters long"
+        if not len(commit_hash) == 40:
```

Comment:
This will be hard to test. 😅 

Maybe we can remove it entirely. 🤔 I think I wanted to assert that the git query was successful and didn't contain an error message. Checking for the length of the parsed output was the proxy.

When this goes wrong, the important thing is that `commit_hash` is printed so that we know what error was raised by git.